test/helper-all.c

```diff
@@ -63,3 +66,251 @@ void connect_and_create(char *pluginkey)
   free_string(author);
   free_string(license);
 }
+
+struct plugin *helper_get_example_plugin(void)
+{
+  struct plugin *p = MALLOC(struct plugin);
+  struct function *f = MALLOC(struct function);
+
+  if (!p || !f)
+    LOG_ERROR("[test] Failed to alloc mem for example plugin.\n");
+
+  p->key = cstring_copy_string("0123456789ABCDEF");
```

Comment:
would be nice to allocate all these strings on the stack. maybe we can change it to:

```c
p->key = (string) {.str = "0123456789ABCDEF", .length = sizeof("0123456789ABCDEF") - 1}};
```

then we can remove the `free_string` calls in the `helper_free_plugin` function.